about.css

```diff
@@ -0,0 +1,527 @@
+  font-family: 'Lato', sans-serif;
+  font-size: 15px;
+  color: var(--primary_4);
+  margin-top: 5%;
+}
+
+.about-mobile-first-section h4 {
+  display: flex;
+  justify-content: center;
+  font-family: 'Lato', sans-serif;
+  font-size: 14px;
+  color: var(--primary_4);
+  margin-top: 2%;
+  padding-bottom: 10%;
+}
+
+.about-second-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.about-second-section hr {
+  border-bottom: 6px solid var(--primary_2);
+  width: 5%;
+}
+
+.about-second-section p {
+  color: #797474;
+  padding-top: 1%;
+}
+
+.about-second-section .line3 {
+  width: 100%;
+  color: var(--primary_4);
+  height: 20px;
+}
+
+.logo2 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 900;
+  font-size: 25px;
+  justify-content: center;
+  border: 1px solid var(--primary_5);
+  padding: 25px 27%;
+  margin: 3% 0;
+  text-align: center;
+}
+
+.about-third-section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
+  background-color: var(--secondary_1);
+}
+
+.about-third-section h2 {
+  justify-content: center;
+  align-self: center;
+  padding-top: 5%;
+  border-top: 2px solid #808080;
+  width: 100%;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+}
+
+.about-third-section h2::after {
+  content: "";
+  background-color: var(--primary_2);
+  display: block;
+  margin: 0 auto;
+  width: 1.5em;
+  height: 2px;
+  border-radius: 0.5px;
+  margin-top: 0.5em;
+}
+
+.about-third-section p {
+  justify-content: center;
+  padding-top: 1em;
+  text-align: center;
+  margin-left: 50px;
+  margin-right: 50px;
+}
+
+.techrunch-1 {
+  position: relative;
+  margin: 5% auto;
+  width: 20rem;
+  height: 12rem;
+}
+
+.techrunch-1 img {
+  width: 100%;
+  height: 100%;
+}
+
+.techrunch-1::before {
+  position: absolute;
+  display: block;
+  content: "";
+  background-color: rgba(255, 0, 0, 0.477);
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+
+.techrunch-1 span {
+  font-family: 'COCOGOOSE', Arial, sans-serif;
+  font-size: 30px;
+}
+
+.center-item {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  color: var(--secondary_1);
+  font-size: 18px;
+  font-weight: 600;
+  line-height: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.partners {
+  background-color: #424146;
+}
+
+.partners h2 {
+  text-align: center;
+  padding-top: 1em;
+  font-family: 'Lato', sans-serif;
+  font-size: 25px;
+  color: #fff;
+}
+
+.partners h2::after {
+  content: "";
+  background-color: var(--primary_2);
+  display: block;
+  margin: 0 auto;
+  width: 1.5em;
+  height: 2px;
+  border-radius: 0.5px;
+  margin-top: 0.5em;
+}
+
+.partners-logo {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  padding: 15px;
+  filter: opacity(0.5) drop-shadow(0 0 0 #fff);
+}
```

Comment:
- [ ] Good job implementing the color changes required, but in order to match the original design, you should divide the partners logo as it follows:
3 of them in the top row and 2 of them in the bottom row

Behance design:

![image](https://user-images.githubusercontent.com/66967274/142472825-c3a81379-b500-4592-b786-fc277b81ccb3.png)


Your design:

![image](https://user-images.githubusercontent.com/66967274/142472856-d77b2195-c2ef-4075-b5f3-737cbdeede53.png)